Add MEMBER_REPRESENTATION.md (#78)
openjs-foundation/standards: MEMBER_REPRESENTATION.md

# Member Representation

> The following document lists OpenJS Foundation members and their representation to any standard organisations:

Reviewer: ```suggestion
The following document lists OpenJS Foundation members and their representation to any standards organizations:
```

> * <firstname> <lastname>

Reviewer 2: richard and i are delegates to tc39 already, should we be added here now or do separate prs?

Author: Let's do separate PRs for adding individuals

>     * [TC39][1]

Reviewer 2: ```suggestion
    * [TC39][]
```

>   * <firstname> <lastname> (#issue)

Reviewer 3: `<firstname> <lastname>` is a poor fit for names from many cultures.

Maybe `<fullname>`

I believe that term is used in vCard to refer to a non-abbreviated name.

> * <firstname> <lastname>
>     * [TC39][]
>     * ...

Reviewer 4: ```suggestion
* [TC39]
  * <firstname> <lastname> (#issue)
  * ...
```

I think the indexing is the wrong way around here, and rather than focusing on which members are representing us, we should focus on which organisations we are represented in. This would also help clarify that we're enumerating representations of OpenJS in particular, not all representations by members of the Working Group.

I also added an `#issue` link to the template, for the issue requesting and approving the representation.

> [1]: https://github.com/tc39

Reviewer 2: ```suggestion
[TC39]: https://github.com/tc39
```
